components/overflow-group/filter-overflow-group.js

```diff
@@ -0,0 +1,62 @@
+import '../filter/filter.js';
+import { css, html, LitElement } from 'lit';
+import { OVERFLOW_DROPDOWN_CLASS, OverflowGroupMixin } from './overflow-group-mixin.js';
+
+function createFilterItem(node) {
+	const dimensionSets = node.querySelectorAll('d2l-filter-dimension-set');
+	const clones = Array.from(dimensionSets).map((set) => set.cloneNode(true));
+	return clones;
+}
+
+/**
+ * A component that can be used to display a group of filters that will be put into an overflow filter when they no longer fit on the first line of their container
+ * @slot - d2l-filters to be added to the container
+*/
+class OverflowGroupFilter extends OverflowGroupMixin(LitElement) {
+
+	static get styles() {
+		return [super.styles, css`
+			::slotted(d2l-filter) {
+				margin-right: 0.5rem;
+			}
+		`];
+	}
+
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+
+		this.addEventListener('d2l-filter-change', this._handleFilterChange);
+	}
+
+	convertToOverflowItem(node) {
+		const tagName = node.tagName.toLowerCase();
+		if (tagName === 'd2l-filter') return createFilterItem(node);
+	}
+
+	getOverflowMenu() {
+		if (this.overflowMenuHidden) return;
+		const overflowItems = this.dropdownItems ? this.dropdownItems.slice(this.chompIndex) : [];
+
+		return html`
+			<d2l-filter class="${OVERFLOW_DROPDOWN_CLASS}" @d2l-filter-change="${this._handleFilterChange}">
+				${overflowItems}
+			</d2l-filter>
+		`;
+	}
+
+	_handleFilterChange(e) {
+		const target = (e.target.classList && e.target.classList.contains(OVERFLOW_DROPDOWN_CLASS)) ? this : e.target;
```

Comment:
The most complicated piece of this is that when:
a) a `d2l-filter-dimension-set-value` (NOT in the overflow filter) changes its `selected` status, the overflow menu version needs to be updated (mutation observer needs to be triggered). In `filter` the list item `selected` state gets update but not the original `d2l-filter-dimension-set-value` so this updates that.
b) a `d2l-filter-dimension-set-value` (in the overflow filter) changes its `selected` status, the original `d2l-filter-dimension-set-value` needs to have its `selected` status updated